### PR TITLE
[update][l] Updating the scripts folder

### PR DESCRIPTION
**Pull request by gradedSystem**
## Changes made:
- Updated the scripts folder
- Removed the old links
- Added `requirements.txt`

cc @anuveyatsu

### CODE REVIEW

File: scripts/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bs4==4.12.3
```
Comment (anuveyatsu) [inline scripts/requirements.txt]: @gradedSystem I didn't find where bs4 was used in the scripts. Maybe it is a mistake?

File: scripts/process.py [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+       yearly data. So, in order to not mess the process of the data, old data from CHURCH/WHITE source for 2015 monthly is used.
+"""
+import os
 import csv
 import zipfile
-
-import os
-import urllib
-#import dataconverters.xls
-epa_src = "http://www3.epa.gov/climatechange/images/\
```
Comment (anuveyatsu) [inline scripts/process.py:8]: @gradedSystem are we saying that EPA source is not valid anymore? Did they stop providing data?

File: scripts/requirements.txt (hunk 1 above)
Comment (gradedSystem) [reply scripts/requirements.txt]: @anuveyatsu yes it is a mistake I will change that in 10-15 minutes

File: scripts/process.py (hunk 2 above)
Comment (gradedSystem) [reply scripts/process.py:8]: It is a valid source however data only available until 2014 and the original source now deprecated and this one is currently much newer: www.cmar.csiro.au

File: scripts/requirements.txt (hunk 1 above)
Comment (gradedSystem) [reply scripts/requirements.txt]: FIXED @anuveyatsu

**Pull request merged**